<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JanelaCadastroClientes_Gato</class>
 <widget class="QDialog" name="JanelaCadastroClientes_Gato">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>328</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gato</string>
  </property>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>200</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Observação:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_gatoInserir">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>290</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Inserir</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>0</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Gato:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>60</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Pelagem*:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_pelagem">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>60</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="txtEdit_observacao">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>190</y>
     <width>104</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>130</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Castrado:</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>68</width>
        <height>17</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QCheckBox" name="cb_castrado">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>270</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>* Campo obrigatório</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_gatoAlterar">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>290</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Alterar</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
